### ISSUE TITLE
I don't think bedrock server work!

### ISSUE THREAD

**Issue by alphxn** [mattmoody05/discord.mc#1]
You should make a status bot for bedrock as well, since it is the new up coming server and there are next to no good status bots, It will be in your favour if you add bedrock servers aswell!

**Comment by mattmoody05** [comment]
Yeah - bedrock support would be good but the main aim of the bot was for personal use in a discord server, which we only used java servers, possibly will have a look in the future though. Thanks
